From 120a53d9fa03ab294d4130a9391753accca2a224 Mon Sep 17 00:00:00 2001
From: Florian Klink <flokli@flokli.de>
Date: Mon, 30 Mar 2020 17:29:53 -0700
Subject: [PATCH 1296/1852] drm/i2c: adv7511: add missing include to
 drm/drm_probe_helper.h

commit 315d8ed241f8f6e2b2df5bb880b5e1b61a54cb58 from
https://github.com/Xilinx/linux-xlnx.git

Otherwise, the compiler complains about not being able to find the
`drm_helper_hpd_irq_event` symbol:

```
drivers/gpu/drm/i2c/adv7511.c: In function 'adv7511_irq_process':
drivers/gpu/drm/i2c/adv7511.c:450:3: error: implicit declaration of function 'drm_helper_hpd_irq_event' [-Werror=implicit-function-declaration]
  450 |   drm_helper_hpd_irq_event(adv7511->encoder->dev);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~

Signed-off-by: Florian Klink <flokli@flokli.de>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/i2c/adv7511.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/i2c/adv7511.c b/drivers/gpu/drm/i2c/adv7511.c
index 523f15664462..3ebbe76972cf 100644
--- a/drivers/gpu/drm/i2c/adv7511.c
+++ b/drivers/gpu/drm/i2c/adv7511.c
@@ -17,6 +17,7 @@
 #include <drm/drm_crtc_helper.h>
 #include <drm/drm_edid.h>
 #include <drm/drm_encoder_slave.h>
+#include <drm/drm_probe_helper.h>
 
 #include "adv7511.h"
 
-- 
2.31.1

